content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md

```diff
@@ -96,16 +96,10 @@ If you have multiple GPG keys, you need to tell Git which one to use.
 {% data reusables.gpg.list-keys-with-note %}
 {% data reusables.gpg.copy-gpg-key-id %}
 {% data reusables.gpg.paste-gpg-key-id %}
-1. To add your GPG key to your bash profile, run the following command:
-  ```shell
-  $ if [ -r ~/.bash_profile ]; then echo 'export GPG_TTY=$(tty)' >> ~/.bash_profile; \
-    else echo 'export GPG_TTY=$(tty)' >> ~/.profile; fi
+1. To add your GPG key to your `.bashrc` startup file, run the following command:
+  ```console
```

Comment:
```suggestion
  ```bash
```

Reply:
FYI, `console` is just an alias for `ShellSession` :smile: 
https://github.com/github/linguist/blob/c34f887f48d81e2ed42b59825fa5fe12d3918d63/lib/linguist/languages.yml#L6392